Libraries/src/Amazon.Lambda.AspNetCoreServer.Hosting/HostingOptions.cs

```diff
@@ -12,4 +13,9 @@ public class HostingOptions
     /// back to JSON to return to Lambda.
     /// </summary>
     public ILambdaSerializer Serializer { get; set; }
+
+    /// <summary>
+    /// The encoding response options used by the Lambda
```

Comment:
Can you add a sentence that informs the reader that this is used to adjust what content types have their data sent to API Gateway as binary data?